advanced/Scripts/query.sh

```diff
@@ -58,12 +61,68 @@ scanList(){
     esac
 }
 
+# function to count new domain entries since last gravity run
+CountNewDomains() {
+  newdomainentries=$(
+    sqlite3 "${gravityDBfile}" << EOSQL
+      ATTACH '${gravityOLDfile}' AS old;
+      SELECT count(DISTINCT domain) FROM gravity WHERE domain NOT IN (SELECT domain FROM old.gravity);
```

Comment:
No tried, but it might be faster with an additional `distinct`

`SELECT count(DISTINCT domain) FROM gravity WHERE domain NOT IN (SELECT  DISTINCT domain FROM old.gravity);`

Reply:
without the additional distinct:
`time pihole -q -countnew
  [i] 44 new domain(s) found.

real    0m25.470s
user    0m16.324s
sys     0m5.009s
`

with the additional distinct:
` time pihole -q -countnew
  [i] 44 new domain(s) found.

real    1m13.115s
user    0m51.230s
sys     0m4.820s
`
I appreciate that you are reviewing this.

Reply:
Okay, let's forget that.